languages/swift/README.md

```diff
@@ -13,9 +13,9 @@ This area will contain everything needed to launch the Swift track, including:
 Before we publicize requesting contribution for this language, the following steps should be done.
 
 - [ ] Have a kick-off discussion between track maintainers
-- [ ] Fill out the [maintainers.md](./maintainers.md) file (e.g. [C#](../csharp/maintainers.md))
+- [x] Fill out the [maintainers.md](./maintainers.md) file (e.g. [C#](../csharp/maintainers.md))
 - [x] Ensure there is a link to your track's GitHub issues on the [main README.md](../../README.md)
-- [ ] [Write a Concept Exercise implementation guide](../../docs/maintainers/writing-a-concept-exercise-github-issue.md)
+- [x] [Write a Concept Exercise implementation guide](../../docs/maintainers/writing-a-concept-exercise-github-issue.md)
```

Comment:
I don't think this is correct. If I look at [that document](https://github.com/exercism/v3/blob/master/languages/swift/reference/implementing-a-concept-exercise.md), it is still the default (stub) implementation. For an example of what that file can look like, see https://github.com/exercism/v3/blob/master/languages/csharp/reference/implementing-a-concept-exercise.md

```suggestion
- [ ] [Write a Concept Exercise implementation guide](../../docs/maintainers/writing-a-concept-exercise-github-issue.md)
```